core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -1760,6 +1760,12 @@ paths:
           required: true
           schema:
             type: string
+        - name: useCache
+          in: query
+          description: True to read entity statistics from cache (eventually consistent), false to force latest statistics (slower). First time reading from the cache may force latest statistics. Default is false.
+          schema:
+            type: boolean
+            default: false
```

Comment:
nit: since the code now checks if the cache is up to date, you can remove the "eventually consistent" comment